### ISSUE TITLE
elogind and eudev

### ISSUE THREAD

**Issue by udeved** [gentoo/eudev#145]
If elogind is used in combination with eudev-3.2.2, an udev error is logged.

> May 8 08:00:55 manjaro kernel: udevd[596]: RUN{builtin}: 'uaccess' unknown /usr/lib/udev/rules.d/73-seat-late.rules:15

I quote from another user who reported the same issue:

https://github.com/systemd/systemd/blob/master/src/udev/udev-builtin.c#L45

> 
> for eudev one would need to get the uaccess code in and then you need to add the --acl option
> to eudev .. just not sure eudev upstream want this.

https://github.com/elogind/elogind/issues/10#issuecomment-300882365

**Comment by Yamakuzure** [comment]
The code needed isn't much, it just updates user ACLs on devices.

See:
https://github.com/systemd/systemd/blob/master/src/udev/udev-builtin-uaccess.c

The offending rule that triggers the error is this one:
https://github.com/elogind/elogind/blob/master/src/login/73-seat-late.rules.in#L15

**Comment by blueness** [comment]
I've resisted putting acl's into eudev, but it looks like you guys need it.  I'll play with it a bit and try to get something out in v3.3

**Comment by CameronNemo** [comment]
FYI I placed a small bounty on this bug, thanks for your work @blueness https://www.bountysource.com/issues/45118863-elogind-and-eudev

**Comment by josch09** [comment]
maybe this guy was wrong...
https://forum.manjaro.org/t/openrc-and-73-seat-late-rules-why-deleting-it-was-a-good-thing/24599

**Comment by Yamakuzure** [comment]
Well... if printing an error by openrc did crash his broadcom wl driver, there is something seriously wrong with his machine.
That rule failing can't likely be the cause. Removing that file only stopped openrc to print an error.

And for the record, I am using the broadcom wl driver ("broadcom-sta"), too, and it doesn't segfault on my machine. (But then, there might be differences between the broadcom-sta ebuild install and the Manjaro dmks install? No idea...)

Having correct ACLs on device nodes is a nice-to-have feature, but I am still not convinced, that it is *that* mission critical. (Maybe I just know too little about it...)

**Comment by CameronNemo** [comment]
@arthurt has proposed a branch for elogind to do the ACL changes itself, so eudev likely will not need to include the uaccess helper to play nice with elogind once that is merged and released

**Comment by blueness** [comment]
@CameronNemo Okay, let me know if there's anything I need to do on my end.  I like this design direction.

**Comment by CameronNemo** [comment]
This issue has been fixed in the recent release of elogind. Should this issue be closed?

If so, @arthurt you can claim the $8 bounty for this issue!

https://github.com/elogind/elogind/issues/10#issuecomment-371775661

**Comment by blueness** [comment]
yeah this can be closed.

**Comment by arthurt** [comment]
> If so, @arthurt you can claim the $8 bounty for this issue!

Can I claim it please, thanks!
